docs/README.md

```diff
@@ -1,51 +1,110 @@
 Welcome to the bcoin docs!
 
 ## Getting Started
-- [Getting Started][getting-started]
-- [Configuration][configuration]
-- [Wallet System][wallet-system]
-- [Design][design]
-- [Guides][guides]
+- [Getting Started](Beginner's-Guide.md)
+- [Configuration](Configuration.md)
+- [Wallet System](Wallet-System.md)
+- [Design](Design.md)
+- [Guides](https://bcoin.io/guides.html)
 
 ## Running
-- [Bcoin CLI][cli]
-- [Running in the Browser][browser]
-- [REST and RPC API][rest-rpc]
+- [Bcoin CLI](CLI.md)
+- [Running in the Browser](https://bcoin.io/guides/browser.html)
+- [REST and RPC API](https://bcoin.io/api-docs/index.html#introduction)
 
 ## Code Examples
-- [Simple Fullnode][example-simple-fullnode]
-- [Connect to Peer][example-connect-peer]
-- [Connecting to the P2P Network][example-p2p]
-- [Creating a Blockchain and Mempool][example-blockchain]
-- [Wallet with Dummy TX][example-wallet-dummy]
-- [Fullnode Object][example-fullnode-wallet]
-- [SPV Sync][example-spv]
-- [Plugin Example][example-peers-plugin]
-- [Client API Usage][example-client-api]
-- [Miner with WorkerPool][example-miner-configs]
-- [Create and Sign TX][example-tx-create-sign]
-- [Get Transaction from Chain][example-tx-from-chain]
-
-
-[getting-started]: Beginner's-Guide.md
-[configuration]: Configuration.md
-[design]: Design.md
-[wallet-system]: Wallet-System.md
-[guides]: http://bcoin.io/guides.html
-
-[cli]: CLI.md
-[browser]: Running-in-the-browser.md
-[rest-rpc]: http://bcoin.io/api-docs/index.html#introduction
-
-[example-p2p]: Examples/connect-to-the-p2p-network.js
-[example-blockchain]: Examples/create-a-blockchain-and-mempool.js
-[example-fullnode-wallet]: Examples/fullnode-and-wallet.js
-[example-spv]: Examples/spv-sync-wallet.js
-[example-wallet-dummy]: Examples/wallet.js
-[example-peers-plugin]: Examples/peers-plugin.js
-[example-client-api]: Examples/client-api.js
-[example-miner-configs]: Examples/miner-configs.js
-[example-connect-peer]: Examples/connect-to-peer.js
-[example-simple-fullnode]: Examples/fullnode.js
-[example-tx-create-sign]: Examples/create-sign-tx.js
-[example-tx-from-chain]: Examples/get-tx-from-chain.js
+
+These code examples are designed to demonstrate how to integrate bcoin modules
+with minimal configuration. They can all be run from the command line without
+interaction and produce some kind of meaningful output. All examples run in
+memory (including databases) to keep the user's disk clean. They mostly run in
+`regtest` mode with a few that connect to the live `main` or `testnet` networks.
+Scripts will either terminate on their own or can be canceled with `ctrl-c`.
+
+Example usage, unless otherwise specified:
+
+```
+$ node Examples/wallet.js
+```
+
+- [Simple Fullnode](Examples/fullnode.js)
```

Comment:
The formatting of these is a bit awkward. 

It could be something such as:
- [Simple Fullnode](Examples/fullnode.js) - Creates a `FullNode` object and connects to `testnet`.
- [Connect to Peer](Examples/connect-to-peer.js) - Connects to a user-defined peer in `regtest` mode.
- [Connecting to the P2P Network](Examples/connect-to-the-p2p-network.js) - Creates `chain`, `pool`, and `mempool` objects for both main and testnet networks.

And gives a better table of contents like overview.